components/patient-data/impl/src/main/resources/ApplicationResources.properties

```diff
@@ -205,6 +206,7 @@ phenotips.UIXField.dateOfBirth=Date of birth
 phenotips.UIXField.dateOfDeath=Date of death
 phenotips.UIXField.lifeStatus=Life status
 phenotips.UIXField.diagnosisNotes=Additional comments
+phenotips.UIXField.geneComments=Gene comments
```

Comment:
@marta- Should this be `Gene comments` or `Genetic comments` or `Genetic data notes` or...?